### DISCUSSION TITLE
change texts and icon in button in checkout process

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3065] [Q&A]
Hello!

I would like to use pretix on behalf of a customer and I would like to change some texts and icons that appear at the button during the checkout process in the frontend. For example I would like to change the text "Add to cart" that appears when you buy a ticket and also hide the cart icon inside the button. Is this possible and if so, what will happen in future updates ?
Will a future update overlap these changes or is there a way to bypass ?
Please consider that the same changes wil have to be done both in english and greek language as well.

Thank you in advance

**Comment by raphaelm** [comment]
Changing languages and texts is not supported. You can do so by changing the code and translation files, but you're on your own regarding future updates. pretix is not built to support this level of customization.
